dev/sbt-plugin/src/sbt-test/sbt-plugin/run-ssl/build.sbt

```diff
@@ -6,8 +6,7 @@ scalaVersion := sys.props.get("scala.version").getOrElse("2.12.6")
 
 interactionMode in ThisBuild := com.lightbend.lagom.sbt.NonBlockingInteractionMode
 
-lagomServiceHttpPort := 7080
```

Comment:
I was under the impression these ports were being configured explicitly so that we can match them in `test` script.

Reply:
Yeah. But I dropped it so that it can default to the hash of the project name and, importantly, to default to `-1` if enabled ssl isn't set.